src/activities/Elsa.Activities.Http/Elsa.Activities.Http.csproj

```diff
@@ -15,26 +15,19 @@
         </Description>
         <PackageTags>elsa, workflows, http</PackageTags>
     </PropertyGroup>
-
-    <ItemGroup Condition=" '$(TargetFramework)' == 'net5.0'">
-        <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="5.0.10" />
-    </ItemGroup>
-
-    <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.1'">
-        <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="3.1.19" />
-    </ItemGroup>

```

Comment:
Will the removal of these references not break applications targeting .NET 5 and .NET 3.1?